bin/compile

```diff
@@ -143,7 +144,26 @@ export CPPPATH="$INCLUDE_PATH"
 export PKG_CONFIG_PATH="$BUILD_DIR/.apt/usr/lib/x86_64-linux-gnu/pkgconfig:$BUILD_DIR/.apt/usr/lib/i386-linux-gnu/pkgconfig:$BUILD_DIR/.apt/usr/lib/pkgconfig:$PKG_CONFIG_PATH"
 
 #give environment to later buildpacks
-export | grep -E -e ' (PATH|LD_LIBRARY_PATH|LIBRARY_PATH|INCLUDE_PATH|CPATH|CPPPATH|PKG_CONFIG_PATH)='  > "$LP_DIR/export"
+topic "Exporting environment"
+BUILDPACK_EXPORT_FILE="$BP_DIR/export"
+export | grep -E -e ' (PATH|LD_LIBRARY_PATH|LIBRARY_PATH|INCLUDE_PATH|CPATH|CPPPATH|PKG_CONFIG_PATH)=' > "$BUILDPACK_EXPORT_FILE"
 
 topic "Rewrite package-config files"
 find "$BUILD_DIR/.apt" -type f -ipath '*/pkgconfig/*.pc' -print0 | xargs -0 --no-run-if-empty -n 1 sed -i -e 's!^prefix=\(.*\)$!prefix='"$BUILD_DIR"'/.apt\1!g'
+
+# post-installation logic for git
+# If git was installed by this buildpack set `GIT_EXEC_PATH` to work around:
+# https://github.com/heroku/heroku-buildpack-apt/issues/137
+if [[ -f "${BUILD_DIR}/.apt/usr/bin/git" ]]; then
+  GIT_CORE_DIR="${BUILD_DIR}/.apt/usr/lib/git-core"
+  if [[ -d "$GIT_CORE_DIR" ]]; then
+    topic "Setting GIT_EXEC_PATH to $GIT_CORE_DIR"
+    # set env var in runtime profile script
+    # shellcheck disable=SC2016
+    echo 'export GIT_EXEC_PATH="$HOME/.apt/usr/lib/git-core"' >> "$RUNTIME_PROFILE_SCRIPT"
+    # export the env var to later buildpacks
+    export | grep -E -e ' (GIT_EXEC_PATH)=' >> "$BUILDPACK_EXPORT_FILE"
```

Comment:
This line needs to be changed since the environment export was removed:

```suggestion
    echo "declare -x GIT_EXEC_PATH=$GIT_CORE_DIR" >> "$BUILDPACK_EXPORT_FILE"
```